tools/c7n_azure/tests_azure/tests_resources/test_defender.py

```diff
@@ -45,3 +45,62 @@ def test_azure_defender_autoprovisioning(self):
         )
         resources = p.run()
         self.assertEqual(len(resources), 1)
+
+    def test_azure_defender_assessments(self):
+        p = self.load_policy({
+            'name': 'test-security-assessments',
+            'resource': 'azure.defender-assessment',
+            'filters': [{
+                'type': 'value',
+                'key': 'properties.status.code',
+                'op': 'eq',
+                'value': 'Healthy'}]})
+        resources = p.run()
+
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['name'], '2a548cf9-6de3-491e-b24a-cf277fe36d4d')
+
+    def test_azure_defender_contacts(self):
+        p = self.load_policy({
+            'name': 'test-security-contacts',
+            'resource': 'azure.defender-contacts'
+        })
+        resources = p.run()
+        self.assertEqual(1, len(resources))
+        self.assertEqual('azurebilling@epmcseclab.com', resources[0]['properties']['email'])
+
+    def test_azure_defender_contacts_validate_schemas(self):
+        p = self.load_policy({
+            'name': 'test-security-contacts-schema-validate',
+            'resource': 'azure.defender-contacts',
+        }, validate=True)
+        self.assertTrue(p)
+
+    def test_azure_defender_jit_policies_query(self):
+        p = self.load_policy({
+            'name': 'test-security-jit-policies',
+            'resource': 'azure.defender-jit-policies',
+        })
+        resources = p.run()
+
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(resources[0]['name'], 'VVvm1')
+
+    def test_azure_defender_jit_policies_filter(self):
+        p = self.load_policy({
+            'name': 'test-security-jit-policies-filter',
+            'resource': 'azure.defender-jit-policies',
+            'filters': [{
+                'type': 'defender-jit-policies-filter',
+                'key': 'properties.virtualMachines[].ports[].number',
+                'op': 'eq',
+                'value': 22
+            }, {
+                'type': 'defender-jit-policies-filter',
+                'key': 'properties.virtualMachines[].id',
+                'op': 'regex',
+                'value': r'\/.+\/virtualMachines\/.+'}]})
```

Comment:
These are good sample use cases, though I'd lean toward covering/documenting them with generic filters rather than adding a custom one.

```suggestion
        p = self.load_policy({
            'name': 'test-security-jit-policies-filter',
            'resource': 'azure.defender-jit-policies',
            'filters': [{
                'type': 'value',
                'key': 'properties.virtualMachines[].ports[].number',
                'op': 'contains',
                'value': 22,
            }, {
                'type': 'list-item',
                'key': 'properties.virtualMachines[]',
                'attrs': [{
                    'type': 'value',
                    'key': 'id',
                    'op': 'regex',
                    'value': r'\/.+\/virtualMachines\/.+',
                }],
            }]
        })
```

Reply:
Hi, thank you for your review. I removed `azure.defender-jit-policies-filter` and made that other resources singular. Also I kept your version of the test but i'm not sure whether it's needed now